Patch-Source: https://github.com/Igalia/cog/commit/44f669f69a77f8a64bc9feb85b909ff1aafc9541
--
From 44f669f69a77f8a64bc9feb85b909ff1aafc9541 Mon Sep 17 00:00:00 2001
From: James Hilliard <james.hilliard1@gmail.com>
Date: Sun, 17 Mar 2024 16:12:12 -0600
Subject: [PATCH] Add support for weston 14 protocols

Weston 13 was released recently and the weston development branch is
now using libweston-14-protocols.

Signed-off-by: James Hilliard <james.hilliard1@gmail.com>
(cherry picked from commit e13c995c5b737bad21fce80fec318d2fd9f34b60)
---
 platform/wayland/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/platform/wayland/meson.build b/platform/wayland/meson.build
index 6d22345a..eca051d3 100644
--- a/platform/wayland/meson.build
+++ b/platform/wayland/meson.build
@@ -48,6 +48,7 @@ wayland_protocols_path = wayland_protocols_dep.get_variable(pkgconfig: 'pkgdatad
 
 if wayland_platform_weston_protocols.length() > 0
     foreach weston_dep_name : [
+            'libweston-14-protocols',
             'libweston-13-protocols',
             'libweston-12-protocols',
             'libweston-11-protocols',
